README.md

```diff
@@ -65,11 +65,11 @@ Kubernetes cluster is set up and the resource aggregator is successfully deploye
 4. On the deployment node, copy the Cisco ACI plugin configuration file and the hook script to `~/plugins/aciplugin`.
 
    ```
-   $ cp install/Kubernetes/helmcharts/aciplugin/aciplugin-config.yaml ~/plugins/aciplugin
+   $ cp PluginCiscoACI/install/Kubernetes/helmcharts/aciplugin/aciplugin-config.yaml ~/plugins/aciplugin
    ```
 
    ```
-   $ cp install/Kubernetes/helmcharts/aciplugin/aciplugin.sh ~/plugins/aciplugin
+   $ cp PluginCiscoACI/install/Kubernetes/helmcharts/aciplugin/aciplugin.sh ~/plugins/aciplugin
    ```
 
```

Comment:
7. Generate the Helm package for the Cisco ACI plugin on the deployment node:
   1. Navigate to `PluginCiscoACI/install/Kubernetes/helmcharts/aciplugin`.
      ```
      $ cd PluginCiscoACI/install/Kubernetes/helmcharts/aciplugin


change it to 
1. Navigate to `PluginCiscoACI/install/Kubernetes/helmcharts/
 cd PluginCiscoACI/install/Kubernetes/helmcharts/

Reply:
Done